myapp/data/raw_query.go

```diff
@@ -100,15 +100,15 @@ func (r *RawQuery) ThreadPreviewQuery(userID int, otherUserID int) (string, stri
 	if len(LatestMessagePreview) > 35 {
 		LatestMessagePreview = LatestMessagePreview[:35] + " . . ."
 	}
-	strLatestMessageTimeSent := LatestMessageTimeSent.Format(time.Kitchen)
+	strLatestMessageTimeSent := LatestMessageTimeSent.Format("Jan 2 3:04PM")
```

Comment:
Not familiar with time formatting, is that a formatter string, or is it hardcoding that date/time?

Reply:
Yes, this is a formatter string. The documentation for time.Format() gives examples of this. The previous one I used, "time.Kitchen", is analogous to "3:04PM".

Reply:
What a strange way to format things. Accepted.